packages/create-block/lib/scaffold.js

```diff
@@ -106,9 +82,41 @@ module.exports = async (
 		style,
 		render,
 		viewScript,
+		variantVars,
 		customPackageJSON,
 		customBlockJSON,
 		example,
+	} );
+
+	const { slug: transformedSlug, namespace: transformedNamespace } =
```

Comment:
`slug` is used later on for messaging and to generate various variations later on. I agree it's not that easily read so I'll refactor.

Reply:
Would it work as expected if the sanitized version gets passed to the transformer? I don't think it matters.

Reply:
It would but there was no guarantee that the slug coming back from the transformer would be formatted correctly. That's the reason we transform and then pull the slug/namespace out afterwards